OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -434,6 +434,9 @@ void MetaModelEditor::addInterfacesData(QDomElement interfaces)
   QDomNodeList subModelList = mXmlDocument.elementsByTagName("SubModel");
   for (int i = 0 ; i < subModelList.size() ; i++) {
     QDomElement subModel = subModelList.at(i).toElement();
+    while(!subModel.firstChildElement("InterfacePoint").isNull()) { //First remove all existing interfaces!
```

Comment:
I think so, that's what I would expect as a user. I can add this functionality and make a new pull request.